src/HealthGPS.Console/model_parser.cpp

```diff
@@ -59,10 +59,31 @@ hgps::RiskFactorSexAgeTable load_risk_factor_expected(const host::Configuration
     return table;
 }
 
+std::unique_ptr<hgps::DummyModelDefinition>
+load_dummy_model_definition(hgps::RiskFactorModelType type, const poco::json &opt) {
```

Comment:
Let's go back to following best practices add a docstring here.

Reply:
The old docstrings in headers vs cpp files debate! See below.